### DISCUSSION TITLE
Examples for relationships

### DISCUSSION THREAD

**Discussion by CaseNewmark** [Azure/azure-mobile-apps#488] [Q&A]
Hi,
can somebody point me to some example code for handling relationships between tables? The Zumo Book (great one btw :)) seems to be a bit outdated on that behalf (not using EntityTableData). However it recommends using relationship tables for 1:many relationships. Is that still the way to go?
Greetings,
Daniel

**Comment by adrianhall** [comment]
I don't have any example code right now.  However, the concepts that the book provides still hold true.  You need to produce a table that - when mapped into Entity Framework - appears as a single table.

I keep getting pulled away from updating the book :-)

**Reply by etcetraw** [reply]
I asked basically the same question on gitter (it's quite there). I think it will be very helpful if we could have a reallife example that involves relationships and inheritance etc.

**Reply by adrianhall** [reply]
Yeah - gitter was not a good channel for support due to lack of usage, so I am closing it down.

**Comment by vigouredelaruse** [comment]
i am also interested to see how you handle ~~odata~~ the associated json patch for arbitrary related entities 

now that i've read the associated code in the sdk i can see this site is useful https://json-gui.esstudio.site/
